app/models/study.rb

```diff
@@ -2281,10 +2283,10 @@ def initialize_cell_metadata(metadata_file, user, opts={local: true})
       Rails.logger.info "#{Time.zone.now}: Validating metadata file headers for #{metadata_file.name}:#{metadata_file.id} in #{self.name}"
       headers = m_file.readline.split(/[\t,]/).map(&:strip)
       @last_line = "#{metadata_file.name}, line 1"
-      second_header = m_file.readline.split(/[\t,]/).map(&:strip)
+      second_header = m_file.readline.split(/[\t,]/).map {|entry| entry.downcase.strip}
```

Comment:
Can you explain this?
|entry| entry.downcase.strip

Reply:
Like the above example of using the `&` block shorthand, this is the expanded version.  Since I need to call two methods (`downcase` to lower-case the string, and then `strip` to remove leading/trailing whitespace).  We're doing this to be more permissive in matching types - so a column in a cluster/metadata file labeled 'Group' instead of 'group' won't cause the values to be stored incorrectly.